src/test/ui/parser/missing_right_square_bracket.rs

```diff
@@ -0,0 +1,3 @@
+// ignore-tidy-trailing-newlines
```

Comment:
Can you please mention the PR as a comment and the issue number in the file name?

Reply:
Added comment and renamed file name.